templates/_layouts/docs.html

```diff
@@ -28,7 +28,7 @@
 
             {% macro side_nav_item(url, title, label, type) -%}
                 <li class="p-side-navigation__item">
-                  <a class="p-side-navigation__link" href="{{ url }}" {% if path == url %}aria-current="page"{% endif %}>
+                  <a class="p-side-navigation__link" href="{{ url }}" {% if (slug and slug in url) or (url == path) %}aria-current="page"{% endif %}>
```

Comment:
Are both parts of this conditional needed? Would only `(slug and slug in url)` work or does it need the `url == path` for some other cases?

Reply:
Initially I had only `slug in url` then highlight side menu item. It was broken for the home page, where slug is `""` making the condition true for all menu items, so everything was highlighted. 

It's not the prettiest condition.. I agree.